code/datums/gamemode/factions/bloodcult/bloodcult_tattoos.dm

```diff
@@ -111,6 +111,10 @@ var/list/blood_communion = list()
 	H.b_eyes = 0
 	H.revive(0)
 	H.status_flags &= ~GODMODE
```

Comment:
Why does this specifically remove godmode?

Reply:
Manifested ghosts have Godmode by default